desk/lib/bolt.hoon

```diff
@@ -63,35 +60,35 @@
     ?>  =(src.bowl our.bowl)
     =/  =bean  !<(bean vase) 
     ?-  -.bean
-        %toggle-which
-      =.  which  switch.bean
-      :-  ~[(emit which/b+which)]
-          this 
+        %toggle-white
+      =.  white  switch.bean
+      :_  this
+      ~[(emit:json white/b+white)]
```

Comment:
I changed this to be consistent with what you do further down -- `:_` instead of `:-` since the head is syntactically larger.